src/Controls/src/Core/PlatformPointerEventArgs.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+namespace Microsoft.Maui.Controls;
+
+/// <summary>
+/// Platform-specific arguments associated with the PointerEventArgs
+/// </summary>
+public class PlatformPointerEventArgs
+{
+#if IOS || MACCATALYST
+	/// <summary>
+	/// UIView that has the attached PointerGestureRecognizer
```

Comment:
Missing cref and sentence should start with "gets"
https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property

Reply:
I think maybe intellisense was having some issues on VSMac with the platform specific classes in the crefs?
![image](https://github.com/dotnet/maui/assets/50846373/fbf4a6cc-9446-4b90-b67e-1da79742c592)

Reply:
Was having issues identifying 'UIKit' on that line too so not sure if this is okay

Reply:
Need to double check this with @davidbritch . Intellisense resolves this on VS Windows if the platform changes to iOS. Otherwise it fails. 
![Switching frameworks](https://github.com/dotnet/maui/assets/32918747/6f0cebdd-88f5-4c43-bcc1-1de7c84dd9b4)
(ignore the fact that I just plopped those docs into Brush - I just wanted to see what happens on Windows)

Reply:
Keep in mind that if you choose to to add the `<remarks>` linking to each native view, the links thatou have to add have to be generic otherwise the docs-tooling blows up. In other words, the link has to be https://learn.microsoft.com/dotnet/api/uikit.uigesturerecognizer and NOT  https://learn.microsoft.com/en-us/dotnet/api/uikit.uigesturerecognizer?view=xamarin-ios-sdk-12. 

If you want to play it safe feel free to simply not include the remarks 👍